src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -199,6 +199,12 @@ void handleDirectCommand(Channel channel, User sender, String payload) {
             return;
         }
 
+        m = Pattern.compile("archive (?:github )repo (\\S+) and prefix repo description with (\\S+)",CASE_INSENSITIVE).matcher(payload);
```

Comment:
```suggestion
        m = Pattern.compile("archive (?:github )repo (\\S+) (\\S+)",CASE_INSENSITIVE).matcher(payload);
```
Something like that IIUC?